src/App.tsx

```diff
@@ -1,11 +1,31 @@
 import React from "react";
 import Example from "./components/Example";
 import Layout from "./components/layout/Layout";
+import PollutedLocationCard from "./components/pollutedLocationCard/PollutedLocationCard";
+import PollutedLocationDTO from "./types/backEnd/PollutedLocationDTO";
+
+type PollutedLocation = PollutedLocationDTO & {
+  title?: string;
+};
```

Comment:
If you define the type in the card component, you can export it here and import it here – you won't need to write duplicate code to define the type